lms/services/organization.py

```diff
@@ -29,6 +29,16 @@ def get_by_public_id(self, public_id: str) -> Optional[List]:
 
         return self._organization_search_query(public_id=public_id).one_or_none()
 
+    def get_hierarchy_root(self, id_) -> Organization:
```

Comment:
Maybe the type annotation here for `id_` will be helpful to distinguish between the pk and public id (although we usually hint that also with the name, but the more the merrier)